travis/test_server.py

```diff
@@ -204,7 +233,9 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
         server_options = []
     print("\nCreating instance:")
     try:
-        subprocess.check_call(["createdb", db])
```

Comment:
What about `subprocess.check_call(["createdb", db], env=os.environ)`?
To send all environment variables avoiding a wrapper `prep_pg_environ` method.